Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1208,6 +1208,111 @@ def get_incidents_command(client, args):
     )
 
 
+def get_process_context(alert, process_type):
+    process_context = {
+        'Name': alert.get(f'{process_type}_process_image_name'),
+        'MD5': alert.get(f'{process_type}_process_image_md5'),
+        'SHA256': alert.get(f'{process_type}_process_image_sha256'),
+        'PID': alert.get(f'{process_type}_process_os_pid'),
+        'CommandLine': alert.get(f'{process_type}_process_command_line'),
+        'Path': alert.get(f'{process_type}_process_image_path'),
+        'Start Time': alert.get(f'{process_type}_process_execution_time'),
+        'Hostname': alert.get('host_name')
+    }
+
+    remove_nulls_from_dictionary(process_context)
+
+    # If the process contains only 'HostName' , don't create an indicator
+    if len(process_context.keys()) == 1 and 'Hostname' in process_context.keys():
+        return {}
+    return process_context
+
+
+def add_to_ip_context(alert, ip_context):
+    action_local_ip = alert.get('action_local_ip')
+    action_remote_ip = alert.get('action_remote_ip')
+    if action_local_ip:
+        ip_context.append(
+            {
+                'Address': action_local_ip
+            }
+        )
+
+    if action_remote_ip:
+        ip_context.append(
+            {
+                'Address': action_remote_ip
+            }
+        )
+
+
+def create_context_from_network_artifacts(network_artifacts, ip_context):
+    domain_context = []
+
+    if network_artifacts:
+        for artifact in network_artifacts:
+            domain = artifact.get('network_domain')
+            if domain:
+                domain_context.append(
+                    {
+                        'Name': domain
+                    }
+                )
+
+            network_ip_details = {
+                'Address': artifact.get('network_remote_ip'),
+                'GEO.Country': artifact.get('network_country')
```

Comment:
make sure it parses correctly in the context and not being exported as a context key with a dot (I think it should be fine).
```suggestion
                'GEO.Country': artifact.get('network_country'),
```